### PR TITLE
Update star operator instructions

### PR DESCRIPTION
**Pull request by msutherl**

### CODE REVIEW

File: getting-started/operating-a-star.udon [hunk 1]
```diff
@@ -29,17 +29,15 @@ Below are some handy operations for the use and maintenance of your star.
...
-+ The dojo will prompt you to enter something. Enter your IP address here.
+Previously star operators needed to run `:dns|ip` in the Dojo in order to set up DNS routing with their parent galaxy. This is no longer required.
 
 ### Maintaining Connectivity
 
-Occasionally, your star might lose network connectivity due to a known timer issue. When this happens, enter the `|bonk` command in the Dojo to reset the timer and restore connectivity.
+Occasionally, your star might lose network connectivity due to a known timer issue or by falling out of sync with the Ethereum blockchain. When this happens, there are a few things to try:
+
+1. Enter the `|bonk` command in the Dojo to reset the timer and restore connectivity.
+2. Run `|start %eth-manage`, then `:eth-manage %look-ethnode` to ensure that your star is synced up to the latest block.
```
Comment (Fang-) [inline getting-started/operating-a-star.udon]: Wondering if it's truly ok to pull people out of the "get eth state from sponsor" flow and into "listen to a central eth node".

`:eth-manage [%look-ship (sein:title our now our)]` should be enough to just kick the subscription, and will generally fix things as long as the sponsor has correct eth state.

Comment (msutherl) [reply getting-started/operating-a-star.udon]: Agree this is better – I don't fully understand how this works!

Comment (rmariani) [reply getting-started/operating-a-star.udon]: Could you incorporate this?

Comment (msutherl) [reply getting-started/operating-a-star.udon]: @Fang- is %look-ship even a function? https://github.com/urbit/arvo/blob/fb65f6e76dd808616faeb6461e6ecdcf52687dc9/app/eth-manage.hoon

Comment (Fang-) [reply getting-started/operating-a-star.udon]: Whoops no, [it's `%look-kick`](https://github.com/urbit/arvo/blob/fb65f6e76dd808616faeb6461e6ecdcf52687dc9/app/eth-manage.hoon#L40).

**Comment by msutherl** [comment]
@rmariani we'll have to create a new PR for this after `zola-port` is merged

**Pull request closed without merging**